GovUk.Frontend.ExampleApp/wwwroot/QuestionsAndAnswersData/searchResults.json

```diff
@@ -0,0 +1,24 @@
+{
+  "results": [
+    {
+      "key": "example1.json",
```

Comment:
This is confusing because, having read your page, I don't think these keys are filenames. If they matched the `key` of `example1`, `example2` etc in your other JSON it would be clearer.

Reply:
I'm a bit confused by your suggestion here. I have made all of the keys end in `.json` in `example1` etc and in `searchResults.json`. For the example app, the keys need to end in `.json` as (annoyingly) the browser returns a 404 when you request `https://localhost:44326/SearchResultsData/example1`